integration-tests/docker-compose-configs/anchor-platform-allowlist/docker-compose-config.override.yaml

```diff
@@ -0,0 +1,43 @@
+# Configuration Description - enable to omnibusAllowList feature to check if an account is whitelisted for SEP-10
```

Comment:
It seems like the override files are largely identical, with the exception of some names and the command that invokes the e2e test itself. Does it make sense to move some of the overrides into the base config?

Reply:
added in c6ef400

Reply:
@erika-sdf i just realized that i misunderstood your comment and didn't quite address this. most of the identical values in the override files are port values, i put those in there because we may want to run all tests in parallel on the same machine at some point and we would need to configure each config override to use a different sets of ports so they dont conflict.